src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline

```diff
@@ -101,12 +91,12 @@ public class AdExchangeBuyerExample {
   }
 
   public AdExchangeBuyer createService() throws IOException, GeneralSecurityException {
-    GoogleCredential credential = GoogleCredential.getApplicationDefault();
-
-    if (credential.createScopedRequired()) {
-      credential =
-          credential.createScoped(Arrays.asList("https://www.googleapis.com/auth/cloud-platform"));
-    }
+    // TODO: Change placeholder below to generate authentication credentials.
```

Comment:
I've complied in spirit - I moved some values from below to above this TODO to separate the content from the top of the method without just leaving a blank line...

Let me know if what I've done is ok with you